Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -189,24 +185,16 @@ struct KalmanFitterResult {
 ///
 /// The void components are provided mainly for unit testing.
 template <typename propagator_t, typename updater_t = VoidKalmanUpdater,
-          typename smoother_t = VoidKalmanSmoother,
-          typename outlier_finder_t = VoidOutlierFinder,
-          typename calibrator_t = VoidMeasurementCalibrator>
+          typename smoother_t = VoidKalmanSmoother>
 class KalmanFitter {
- public:
-  /// Shorthand definition
-  using MeasurementSurfaces = std::multimap<const Layer*, const Surface*>;
```

Comment:
Was this unused?

Reply:
Yes, probably a left-over from a previous implementation.